odoo/tests/common.py

```diff
@@ -173,14 +173,6 @@ def records(self):
 # Main classes
 # ------------------------------------------------------------
 class OdooSuite(unittest.suite.TestSuite):
-
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        from odoo.http import root
-        if not root._loaded:
-            root.load_addons()
-            root._loaded = True
```

Comment:
I would expect this was added because running tests doesn't necessarily imply that the HTTP server was started. That is likely still the case.

Reply:
I think it is no longer necessary since now it loads in initialize_sys_path.